common/lib/capa/capa/templates/choicetext.html

```diff
@@ -59,10 +59,12 @@
         <span id="answer_${id}"></span>
     </fieldset>
         <input class= "choicetextvalue" type="hidden"  name="input_${id}{}" id="input_${id}" value="${value|h}" />
-    
+
     <div class="indicator-container">
     % if input_type == 'checkbox' or not element_checked:
-        <span class="status ${status.classname}" id="status_${id}"></span>
+        <span class="status ${status.classname}" id="status_${id}">
+            <span class="sr">${_(status.classname)}</span>
```

Comment:
@alisan617 

This line is what is causing your python failures I believe. I believe you want the following 

`_(${status.classname})`

I am double checking that though